packages/connector-schemas/src/error.ts

```diff
@@ -0,0 +1,24 @@
+export enum ConnectorErrorCodes {
```

Comment:
we prefer the same format as i18n keys for error codes since they are extendable.

Reply:
These error codes will be handled by a connector error handler middleware (substitute by i18n keys).

Reply:
Though is not the same as i18n, it can still be optimized, `enum` is not good for extend. Can do it later.